fs_product_multi_image/views/product_template.xml

```diff
@@ -20,17 +20,20 @@
             </field>
             <page name="sales" position="after">
                 <page name="images" string="Images">
-                    <field name="image_ids" widget="fs_image_relation_dnd_upload">
+                    <field
+                        name="image_ids"
+                        widget="fs_image_relation_dnd_upload"
+                        mode="kanban"
+                    >
                         <tree>
                             <field name="sequence" widget="handle" />
                             <field name="name" />
+                            <field name="link_existing" invisible="1" />
+                            <field name="image_id" invisible="1" />
+                            <field name="image" invisible="1" />
+                            <field name="specific_image" invisible="1" />
                         </tree>
```

Comment:
Is-it the right way to define the kanban?

Reply:
I added a specific kanban view for the image relation model so we don't need to define an inline kanban view here (if this is what you asked).